src/util/util.js

```diff
@@ -0,0 +1,10 @@
+const $ = (selector) => document.querySelector(selector);
```

Comment:
```suggestion
const $ = (selector, node = document) => node.querySelector(selector);
```
이렇게 확장해보는건 어떨까요??? querySelector는 꼭 document에서 사용해야하는건 아니에요:)

Reply:
좋습니다. 어떻게 잘 사용할 수 있을지 고민해서 적용해보겠습니다~